src/Files.App/Helpers/QuickLookHelpers.cs

```diff
@@ -64,7 +64,6 @@ static async Task<int> QuickLookServerAvailable()
 		try
 		{
 			var result = await QuickLookServerAvailable();
-			App.Logger.Info($"QuickLook detected: {result != 0}");
```

Comment:
I think it should output a log if QuickLook not detected when LaunchQuickLookAction is executed.

Reply:
👍